pages/design/tools/index.mdx

```diff
@@ -1,3 +1,8 @@
 # Design tools
 
+Figma is a vital part of Primer Design's workflow. Figma's collaborative environment allows designers to work together across distance and give feedback asynchronously. Learn more about [GitHub's remote design culture here](https://www.figma.com/blog/github-case-study/).
+
+## Resources
+We provide some open source Figma files to help designers create more with GitHub.com and other GitHub products. Download them below.
+
```

Comment:
Plan to add Figma file links here

Reply:
I wonder if it makes more sense for this content to live on https://primer.style/design/tools/figma? Rather than on the main tools page

Reply:
+1 for @emilybrick's suggestion

Reply:
I think the navigation for Sketch/Figma is a little out-of-the-way, so I think providing direct download links on the front page encourage a little more action. The Figma/Sketch pages can also host them but maybe focus more on the documentation of each component and principles of using them within the files?

This would require a lot more documentation on the design tool pages, but I think it would help give a better purpose to each page.